src/message.rs

```diff
@@ -664,6 +664,12 @@ impl Message {
         self.viewtype
     }
 
+    /// Forces the message to **keep** [Viewtype::Sticker]
+    /// e.g the message will not be converted to a [Viewtype::Image].
+    pub fn force_sticker(&mut self) {
+        self.param.set_int(Param::ForceSticker, 1);
```

Comment:
Wouldn't `ForceViewtype` be better? Of course we can rename it if there will be a need, but still

Reply:
I believe fore_sticker is way more expressive. For force_viewtype I would expect an argument where I can specify a viewtype that I wish for

Reply:
If the same problem will appear in the future, say, for Audio vs Voice, ForceViewtype would be a more common solution. But for now we can leave it ForceSticker of course